yearn/apy/v2.py

```diff
@@ -30,30 +30,27 @@ def closest(haystack, needle):
 def simple(vault, samples: ApySamples) -> Apy:
     harvests = sorted([harvest for strategy in vault.strategies for harvest in strategy.harvests])
 
-    if len(harvests) < 2:
-        raise ApyError("v2:harvests", "harvests are < 2")
-
-    now = closest(harvests, samples.now)
-    week_ago = closest(harvests, samples.week_ago)
-    month_ago = closest(harvests, samples.month_ago)
-    inception_block = harvests[:2][-1]
-
     contract = vault.vault
     price_per_share = contract.pricePerShare
+    now_price = price_per_share(block_identifier=samples.now)
+    inception_price = 10 ** contract.decimals()
+    
+    if len(harvests) < 2 or now_price == inception_price:
+        raise ApyError("v2:harvests", "harvests are < 2")
```

Comment:
can we split these into two if statements with two error messages, otherwise the error description might not be accurate

Reply:
sounds good to me, if you want to make this changes that works :)